packages/pigeon/lib/ast.dart

```diff
@@ -257,6 +257,30 @@ class Enum extends Node {
   }
 }
 
+/// Represents a Enum member.
+class EnumMember extends Node {
+  /// Parametric constructor for [EnumMember].
+  EnumMember({
+    required this.name,
+    this.documentationComments = const <String>[],
+  });
+
+  /// The name of the enum member.
+  String name;
```

Comment:
final String name;

Reply:
Fixed what you pointed out.
I noticed that the existing class `Method` `Api` `NamedType` `Class` `Enum` `Root` in `ast.dart` is in the same.
(not `final`).
Should I fix these as well?